### PR TITLE
📖  Release process for the verifier

### PR DESCRIPTION
**Pull request by laurentsimon**
Release process for the verifier

The hash added in this PR has already been verified in https://github.com/slsa-framework/slsa-github-generator/pull/113/files

### CODE REVIEW

**Comment by laurentsimon** [comment]
friendly ping for review

File: RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+```
+
+3. Update the latest version in the [README.md](./README.md).
+
+4. Send a pull request with the changes. In the description, explain the steps to verify the hash update, i.e., reviewers shoud LGTM only if the provenance verification succeeds
+and the hash in the pull request matches the one computed on the binary. You can use [#slsa-framework/slsa-github-generator#113](https://github.com/slsa-framework/slsa-github-generator/pull/113) as example.
+
+## Update builders
+
+Send a similar pull request to update the hash and version of the verifier for the workflow [slsa-framework/slsa-github-generator/blob/main/.github/workflows/builder_go_slsa3.yml#L30-L31](https://github.com/slsa-framework/slsa-github-generator/blob/main/.github/workflows/builder_go_slsa3.yml#L30-L31). Explain the steps to verify the hash. If the pull request for the verifier is already merged, you can simply point to it instead.
```
Comment (ianlewis) [inline RELEASE.md:70]: nit: these lines are likely to change. Maybe leave them off the url?

Comment (ianlewis) [reply RELEASE.md:70]: Once we have created one we can maybe add a link to this doc as an example.

File: RELEASE.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+1. Compute the hash of the binary. One of the following commands will do:
+```
+$ cat slsa-verifier-linux-amd64.intoto.jsonl | jq -r '.payload' | base64 -d | jq -r '.subject[0].digest.sha256'
+```
+or
+```
+$ sha256sum slsa-verifier-linux-amd64
+```
+
+2. Add an additional entry at the top of [SHA256SUM.md](./SHa256SUM.md):
```
Comment (ianlewis) [inline RELEASE.md:56]:
```suggestion
2. Add an additional entry at the top of [SHA256SUM.md](./SHA256SUM.md):
```

**Review by ianlewis — APPROVED** [review]

File: RELEASE.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+2. Clone the slsa-verifier repo, compile and verify the provenance:
+```
+$ git clone git@github.com:slsa-framework/slsa-verifier.git
+$ cd slsa-verifier
+$ (Optional: git checkout tags/v1.0.0)
+$ go run . -artifact-path slsa-verifier-linux-amd64 -provenance slsa-verifier-linux-amd64.intoto.jsonl -source github.com/slsa-framework/slsa-verifier -tag vX.Y.Z
+```
+
+If the provenance verification fails, abort. Otherwise, continue.
```
[the remark this answers is not in the document]
Comment (laurentsimon) [reply RELEASE.md:41]: done. Thanks

File: RELEASE.md (hunk 1 above)
Comment (laurentsimon) [reply RELEASE.md:70]: They are unlikely to change. I'll keep the link so we at least have a rough line number, even if they change.
Note: We have created v0.0.1 already

Comment (ianlewis) [reply RELEASE.md:70]: no worries.

If we have a PR that exists and roughly follows these instructions then that would be great to add as an example.

Comment (ianlewis) [reply RELEASE.md:70]: I mean for v0.0.1. I didn't see one in a casual search.

**Pull request merged**